provisioning/roles/nginx/templates/nginx.conf.j2

```diff
@@ -0,0 +1,32 @@
+user  nginx;
+worker_processes  {{ worker_processes }};
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  {{ max_clients }};
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
```

Comment:
Oui, je devais faire des expérimentations et j'ai dû testé TCP_CORK mais c'est pas très important, je pense qu'on devrait discuter de comment on veut configurer NGINX pour qu'il soit performant, et to be honest, je ne suis pas un pro ultime de ça. Mais je suis intéressé par le stress testing HTTP et `wrk`.